portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,39 +31,51 @@
 import java.util.List;
 import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static class Comment {
-    private String email, comment;
-    public Comment(String email, String comment){
+    private final String email, comment, time;
+    public Comment(String email, String comment, String time){
       this.email = email;
       this.comment = comment;
+      this.time = time;
     }
   }
-  private List<Comment> comments = new ArrayList<Comment>();
-
+  private Query comments_query = new Query("Comment").addSort("time", SortDirection.DESCENDING);
```

Comment:
should this be final and CONSTANT_CASE?

(alternatively, just inline it in doGet. It looks "cheap" to construct, compared to `datastore`)

Reply:
put it in doGet